### DISCUSSION TITLE
Password

### DISCUSSION THREAD

**Discussion by Wolf2000Pi** [whoopsmonitor/whoopsmonitor#63] [General]
Hi 
The password never fits no matter whether I use CLI or CapRover, they could help me.
Thank you

**Comment by danielrataj** [comment]
Hi, can you share the output of ./docker-compose.yml file from the root of the app (in case of CLI)?

I tried to run a new installation a short moment ago and it works fine.

However, it might help to restart the API container as well, like:

`docker restart CONTAINER_WITH_API`

Thank you.
